xml_converter/test_cases/test_expected_outputs.json

```diff
@@ -0,0 +1,38 @@
+{
+    "can_fade": {
+        "path": "../test_cases/can_fade_tests",
```

Comment:
this json file is in the `test_cases` directory. A relative path from where this file is located would be `can_fade_tests` because the `../test_cases` goes up a folder and then back in.